<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="testAll"
           tests="7"
           failures="1"
           errors="0"
           skipped="6"
           time="0.003">
  <properties />
            <testcase name="initDriver" time="0.000" classname="com.yiibai.primera.testng.cases.NewsDetailsTest">
                <failure type="java.lang.AssertionError"
                  message="expected object to not be null"
              >
<![CDATA[java.lang.AssertionError: expected object to not be null
  org.testng.Assert.fail(Assert.java:94)
  org.testng.Assert.assertNotNull(Assert.java:404)
  org.testng.Assert.assertNotNull(Assert.java:389)
  com.yiibai.primera.testng.cases.NewsDetailsTest.initDriver(NewsDetailsTest.java:23)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.lang.reflect.Method.invoke(Method.java:497)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
  org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:564)
  org.testng.internal.Invoker.invokeConfigurations(Invoker.java:213)
  org.testng.internal.Invoker.invokeConfigurations(Invoker.java:138)
  org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:175)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:107)
  org.testng.TestRunner.privateRun(TestRunner.java:767)
  org.testng.TestRunner.run(TestRunner.java:617)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
  org.testng.SuiteRunner.run(SuiteRunner.java:240)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
  org.testng.TestNG.run(TestNG.java:1057)
  org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
  org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
  org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
      </failure>

          </testcase>
  
            <testcase name="imageSwitch" time="0.0" classname="com.yiibai.primera.testng.cases.NewsDetailsTest">
          <skipped />
    </testcase>
            <testcase name="share" time="0.0" classname="com.yiibai.primera.testng.cases.NewsDetailsTest">
          <skipped />
    </testcase>
            <testcase name="fontSize" time="0.0" classname="com.yiibai.primera.testng.cases.NewsDetailsTest">
          <skipped />
    </testcase>
            <testcase name="comment" time="0.0" classname="com.yiibai.primera.testng.cases.NewsDetailsTest">
          <skipped />
    </testcase>
            <testcase name="collect" time="0.0" classname="com.yiibai.primera.testng.cases.NewsDetailsTest">
          <skipped />
    </testcase>
            <testcase name="like" time="0.0" classname="com.yiibai.primera.testng.cases.NewsDetailsTest">
          <skipped />
    </testcase>
  
  
</testsuite>
